spec.html

```diff
@@ -8684,8 +8684,12 @@ <h1>SetFunctionName ( _F_, _name_ [ , _prefix_ ] )</h1>
           1. Let _description_ be _name_'s [[Description]] value.
           1. If _description_ is *undefined*, set _name_ to the empty String.
           1. Else, set _name_ to the string-concatenation of *"["*, _description_, and *"]"*.
+        1. If _F_ has an [[InitialName]] internal slot, then
+          1. Set _F_.[[InitialName]] to _name_.
         1. If _prefix_ is present, then
           1. Set _name_ to the string-concatenation of _prefix_, the code unit 0x0020 (SPACE), and _name_.
+        1. If _F_ has an [[InitialName]] internal slot, then
+          1. Optionally, set _F_.[[InitialName]] to _name_.
```

Comment:
```suggestion
          1. If _F_ has an [[InitialName]] internal slot, then
            1. Optionally, set _F_.[[InitialName]] to _name_.
```
since this only applies if prefix is present

Reply:
It works either way.

Reply:
You can also omit “Optionally,”:
```suggestion
          1. If _F_ has an [[InitialName]] internal slot, then
            1. Set _F_.[[InitialName]] to _name_.
```

Reply:
I believe that was added explicitly, per today's TC39 consensus, to allow, but not require, `get ` and `set ` prefixes.